packages/dashboard/prisma/schema.prisma

```diff
@@ -143,6 +143,7 @@ model Deployment {
   triggerer    String   @default("Lagon")
   commit       String?
   isProduction Boolean  @default(false)
+  platform     String   @default("CLI")
```

Comment:
The platform should be set to a `Function` and not a `Deployment`

Reply:
Give it a try and see if it meets your requirements.